src/events/components/Attendance/UnattendButton.tsx

```diff
@@ -11,15 +14,22 @@ interface IAttendButtonProps {
 
 const UnattendButton: FC<IAttendButtonProps> = ({ eventId, isOnWaitList, waitListNumber }) => {
   const dispatch = useDispatch();
-  const signOff = () => dispatch(removeAttendeeByEventId(eventId));
+  const [addToast] = useToast({ type: 'success', duration: 5000 });
+  const signOff = async () => {
+    try {
+      await dispatch(removeAttendeeByEventId(eventId)).then(unwrapResult);
+      addToast('Du har blitt meldt av arrangementet');
+    } catch (err) {
+      addToast(`Noe gikk galt under avmeldelse, ERROR: ${err.message}`, { type: 'error' });
```

Comment:
Same as above, I suggest considering if it makes sense to stick to either `async-await` and `try-catch` or just `.then()`

Reply:
I've decided that i think that its more readable with a try catch. I however removed the usage of then, instead i dispatch outside of the try-catch and unwrap it inside of the try-catch